Modules/gcmodule.c

```diff
@@ -1683,6 +1787,70 @@ gc_get_freeze_count_impl(PyObject *module)
 }
 
 
+/*[clinic input]
+gc.enable_object_debugger as gc_py_enable_object_debugger -> NoneType
```

Comment:
I would not use `-> NoneType`. It does not add clarity. Actually `return Py_None` without increfing looks suspicious. It is clearer to use Py_RETURN_NONE.